modern/src/LoginPage.js

```diff
@@ -118,6 +137,12 @@ const LoginPage = () => {
             </div>
           </FormControl>
         </form>
+        {registerDialogShown && <RegisterDialog open={registerDialogShown} onResult={handleRegisterResult} />}
+        <Snackbar
```

Comment:
Please follow existing code formatting style.

Reply:
Also, is it possible to move snackbar to the `RegisterDialog` component?

Reply:
I hope formatting is good now.
I moved snack bar to RegisterDialog.